docs/modules/publisher-dmmigration.md

```diff
@@ -0,0 +1,141 @@
+[Home](../../readme.md)
+
+# Migration path
+
+The latest version 2.8.2 adds support for configuration via direct methods. OPC Publisher implements multiple [IoT Hub Direct Methods](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods) which can be called from an application leveraging the [IoT Hub Device SDK](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks). This document provides the migration path from 2.5.x to 2.8.2.
+
+The direct methods' request payload of version 2.8.2 is backwards compatible with the 2.5.x direct methods. The payload schema allows configuration of additional extensions introduced in the `pn.json` in the publisher 2.6.x and newer e.g. DataSetWriterGroup, DataSetWriterId, QueueSize per node, etc.
+
+For this set of methods, the encoding is JSON and no compression or payload chunking mechanism is applied in order to ensure the backwards compatibility with the 2.5.x version of OPC Publisher module.
+
+
+
+## Direct Methods of version 2.5.x
+
+The following  table describes the direct methods which were available in OPC Publisher 2.5.x with request and response. The direct methods which are removed in version 2.8.2 are `GetDiagnosticLog`, `GetDiagnosticStartupLog`, `ExitApplication` and `GetInfo`.
+
+| 2.5.x                            |                                                              |                                                              |                          |
+| -------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------ |
+| **MethodName**                   | **Request**                                                  | **Response**                                                 | **in 2.8.2** |
```

Comment:
`Request` column listing specific fields and not a JSON looks strange to me. I would still ask for a sample of request payload for `PublishNodes` to be added in the document, similar to what we have for `2.8.2`.